crates/oxc_transformer/src/es2018/options.rs

```diff
@@ -0,0 +1,20 @@
+use serde::Deserialize;
+
+use super::object_spread::ObjectSpreadOptions;
+
+#[derive(Debug, Default, Clone, Deserialize)]
+#[serde(default, rename_all = "camelCase", deny_unknown_fields)]
+pub struct ES2018Options {
+    #[serde(skip)]
+    pub object_spread: Option<ObjectSpreadOptions>,
+    // TODO: Implement object rest
+    // pub object_rest: bool,
```

Comment:
Should be deleted?

Reply:
The `object_rest` has not been implemented yet. How to mark TODO in OXC?